package.json

```diff
@@ -45,7 +45,8 @@
     "jsonc-parser": "~3.2.1",
     "markdownlint": "~0.33.0",
     "minimatch": "~9.0.3",
-    "run-con": "~1.3.2"
+    "run-con": "~1.3.2",
+    "toml": "^3.0.0"
```

Comment:
Please match the `~` prefix used by this project for version references.

Reply:
Changed it. 
Nevertheless it seems to be inconsistent since the devDependencies are using ^

Reply:
I reference all dependencies exactly in my projects; this one is more strict about dependencies than devDependencies.

Reply:
Ok, makes sense. Thanks for the clarification